packages/fxa-settings/src/pages/Signin/Signin/index.test.tsx

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Signin from '.';
+import { usePageViewEvent } from '../../../lib/metrics';
+import { EXAMPLE_EMAIL, EXAMPLE_SERVICE, EXAMPLE_ICON } from './mocks';
+
+jest.mock('../../../lib/metrics', () => ({
+  usePageViewEvent: jest.fn(),
+}));
+
+// TODO: Test l10n
```

Comment:
Could be good to include the blocking issue (FXA-6461 (handle embedded tags)) to know when to enable these tests.

If including the code block on lines 18-19, adding the commented out imports and test line might make enabling these tests faster once FXA-6461 is resolved? Otherwise, maybe just a "TODO: test l10n" would be enough.

```
// import { getFtlBundle, testAllL10n } from 'fxa-react/lib/test-utils';
// import { FluentBundle } from '@fluent/bundle';
```

// testAllL10n(screen, bundle);

Reply:
That's a great idea, I'll add those all in